pallets/elections/src/migrations/v1_to_v2.rs

```diff
@@ -76,3 +76,55 @@ pub fn migrate<T: Config, P: PalletInfoAccess>() -> Weight {
     StorageVersion::new(2).put::<P>();
     T::DbWeight::get().reads(reads) + T::DbWeight::get().writes(writes)
 }
+
+#[cfg(feature = "try-runtime")]
+pub fn pre_upgrade<T: Config, P: PalletInfoAccess>() -> Result<(), &'static str> {
+    match MembersPerSession::get() {
+        Some(_) => {}
+        _ => return Err("No `Members` storage"),
+    }
+    match ReservedMembers::<T>::get() {
+        Some(_) => {}
+        _ => return Err("No `Members` storage"),
+    }
+    match NonReservedMembers::<T>::get() {
+        Some(_) => {}
+        _ => return Err("No `Members` storage"),
+    }
+    match ErasMembers::<T>::get() {
+        Some(_) => {}
+        _ => return Err("No `Members` storage"),
+    }
+
+    if StorageVersion::get::<P>() == StorageVersion::new(1) {
+        Ok(())
+    } else {
+        Err("Bad storage version")
+    }
+}
+
+#[cfg(feature = "try-runtime")]
+pub fn post_upgrade<T: Config, P: PalletInfoAccess>() -> Result<(), &'static str> {
```

Comment:
Suggestion: the same recommendation for passing old state here in order to validate content transfer